src/test/scala/uk/gov/nationalarchives/tdr/api/utils/TestUtils.scala

```diff
@@ -1,186 +1,214 @@
 package uk.gov.nationalarchives.tdr.api.utils
 
-import java.sql.{PreparedStatement, ResultSet, Timestamp}
-import java.time.Instant
-import java.util.UUID
 import akka.http.scaladsl.model.headers.OAuth2BearerToken
 import akka.http.scaladsl.unmarshalling.{FromResponseUnmarshaller, Unmarshaller}
 import akka.stream.Materializer
 import com.tngtech.keycloakmock.api.KeycloakVerificationMock
 import com.tngtech.keycloakmock.api.TokenConfig.aTokenConfig
 import io.circe.Decoder
 import io.circe.parser.decode
-import uk.gov.nationalarchives.tdr.api.db.DbConnection
+import slick.jdbc.JdbcBackend
 import uk.gov.nationalarchives.tdr.api.model.file.NodeType
 import uk.gov.nationalarchives.tdr.api.service.FileMetadataService._
+import uk.gov.nationalarchives.tdr.api.service.FinalTransferConfirmationService._
 import uk.gov.nationalarchives.tdr.api.service.TransferAgreementService._
+import uk.gov.nationalarchives.tdr.api.utils.TestUtils._
 
+import java.sql._
+import java.time.Instant
+import java.util.UUID
 import scala.concurrent.ExecutionContext
 import scala.io.Source.fromResource
 
 //scalastyle:off number.of.methods
-object TestUtils {
-  val defaultFileId: UUID = UUID.fromString("07a3a4bd-0281-4a6d-a4c1-8fa3239e1313")
-
-  private val tdrPort: Int = 8000
-  private val testPort: Int = 8001
-  private val tdrMock: KeycloakVerificationMock = createServer("tdr", tdrPort)
-  private val testMock: KeycloakVerificationMock = createServer("test", testPort)
-
-  private def createServer(realm: String, port: Int): KeycloakVerificationMock = {
-    val mock: KeycloakVerificationMock = new KeycloakVerificationMock(port, "tdr")
-    mock.start()
-    mock
+class TestUtils(db: JdbcBackend#DatabaseDef) {
+  val connection: Connection = db.source.createConnection()
+
+  def deleteTables(): Boolean = {
```

Comment:
Is this still needed, now that the tests are running in individual containers?

Reply:
Unfortunately it is. The tests run in one container per class. I did try it with one container per test but it was far too slow, around 8 minutes running the tests in CI. So now it's one container per class but without major refactoring of everything, we still need this. Maybe one for the future.